### PR TITLE
nvm 0.1.0

### PR DESCRIPTION
**Pull request by Almouro**
Hi guys,

I've been using Bitrise for a few weeks now and it works great! :)
For npm projects like `react-native`, it's often nice to be able to exactly select the node version. This is exactly what [nvm](https://github.com/creationix/nvm) does!

I wanted to try it on `Bitrise.io`, so I set my steplib as a custom lib source to then run this step:
```
---
format_version: 1.2.0
default_step_lib_source: https://github.com/almouro/bitrise-steplib.git
trigger_map:
- pattern: "*"
  is_pull_request_allowed: true
  workflow: primary
workflows:
  primary:
    steps:
    - nvm@0.1.0: {}
app:
  envs:
  - opts:
      is_expand: false
    BITRISE_PROJECT_PATH: ios-simple-objc/ios-simple-objc.xcodeproj
  - opts:
      is_expand: false
    BITRISE_SCHEME: ios-simple-objc
```
It runs great, but the UI workflow editor will not work, saying *There’s an issue in your bitrise.yml configuration. You have to fix it before you could use the UI editor again. 
Error: Cannot read property 'versions' of undefined.*
Any idea why? First time I create a step, sorry if I missed something! :[

### New Pull Request Checklist

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `stepman audit --step-yml ./step.yml` (in the step's repository - note, `stepman` is part of the Bitrise CLI, you don't have to install it separately)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by viktorbenei** [comment]
Awesome, thanks a bunch!

Re: "Error: Cannot read property 'versions' of undefined." - you get this in the editor because there's no "nvm" step **yet** in the library, but this will change soon ;)

We'll do a review asap - first step review takes a bit longer, as we try to advise changes which might be hard to correct later (without breaking compatibility); subsequent version/update shares are way faster (usually just a couple of hours).

File: steps/nvm/0.1.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+source_code_url: https://github.com/almouro/bitrise-nvm-step
+support_url: https://github.com/almouro/bitrise-nvm-step/issues
+published_at: 2016-07-02T22:15:30.551754569+02:00
+source:
+  git: https://github.com/Almouro/bitrise-nvm-step.git
+  commit: 39aa1b7d154d4498b16a5eb6a0f9d13bc479d62c
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
```
Comment (viktorbenei) [inline steps/nvm/0.1.0/step.yml]: I'd remove the `ios` project type filter, this step can work with other types of projects too

File: steps/nvm/0.1.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  See https://github.com/creationix/nvm
+website: https://github.com/almouro/bitrise-nvm-step
+source_code_url: https://github.com/almouro/bitrise-nvm-step
+support_url: https://github.com/almouro/bitrise-nvm-step/issues
+published_at: 2016-07-02T22:15:30.551754569+02:00
+source:
+  git: https://github.com/Almouro/bitrise-nvm-step.git
+  commit: 39aa1b7d154d4498b16a5eb6a0f9d13bc479d62c
+host_os_tags:
+- osx-10.10
```
Comment (viktorbenei) [inline steps/nvm/0.1.0/step.yml]: Does it work on linux (Android stack) too? I checked the code, and I believe this should work there too. Did you try it?

**Comment by viktorbenei** [comment]
Btw you can test & use your step before sharing as described in the step template https://github.com/bitrise-steplib/step-template#how-to-create-your-own-step

**Comment by godrei** [comment]
Hi @Almouro ,
would you like to fix, the mentioned issues, or should we merge your step?
The issues are not critical, so we can merge it, but it would be great to fix them in the next version of the step.
Thanks for the contribution! 🚀

**Comment by viktorbenei** [comment]
We decided to merge this version, thank you for the contribution!

Feel free to publish an update once you have the time for adding the linux support, or if you want to add anything else to the step ;)

**Pull request merged**